CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Codeowners for IREE Github Repository.
+# Refer to https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+# for syntax of this file.
+
+# Global code owners
+* benvanik
+
+# Code owners for individual components/directories
+/iree/compiler/Codegen/ MaheshRavishankar
```

Comment:
Usernames should have leading `@` symbols

here are some samples: https://cs.github.com/?scopeName=All+repos&scope=&q=path%3A.github%2FCODEOWNERS (if you're in the github code search preview)

Reply:
Thanks. missed that